src/card/card.jsx

```diff
@@ -13,16 +13,42 @@ const Card = React.createClass({
   },
 
   propTypes: {
+    /**
+    * Whether a click on this card component expands the card. Can be set on any child of the Card component.
```

Comment:
one more thing, sorry :grin: 

jsdoc better be formatted like this:

```js
/**
 * Foo
 */
```
instead of

```js
/**
* Foo
*/
```
that extra indentation.